src/components/CardSelector/EmptyCardSelector/EmptyCardSelector.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import clsx from 'clsx';
+import './EmptyCardSelector.scss';
+import {EmptyCardSelectorProps} from './EmptyCardSelector.types';
+import {Typography} from '~/components';
+
+const EmptyCardSelectorForwardRef: React.ForwardRefRenderFunction<HTMLButtonElement, EmptyCardSelectorProps> = ({
+    label,
+    id,
+    className,
+    isDisabled = false,
+    isReadOnly = false,
+    onClick = () => undefined,
+    ...props
+}, ref) => {
+    const handleOnClick: React.MouseEventHandler = e => {
+        if (isDisabled || isReadOnly) {
+            return;
+        }
+
+        onClick(e);
+        (e.currentTarget as HTMLElement).blur();
+    };
+
+    return (
+        <button
+            ref={ref}
+            id={id}
+            type="button"
+            className={clsx('moonstone-emptyCardSelector',
+                (isDisabled || isReadOnly) && 'moonstone-emptyCardSelector_disabled',
+                'flexCol_center',
+                'flexFluid',
+                'alignCenter',
+                className
+            )}
+            aria-disabled={isDisabled || isReadOnly}
+            onClick={e => handleOnClick(e)}
+            {...props}
+        >
+            {label && <Typography className={clsx('moonstone-emptyCardSelector_label')}>{label}</Typography>}
```

Comment:
From jcontent usage, looks like we make use of both icon and label for the empty card display. Is it possible to have them here as well? https://github.com/Jahia/jcontent/blob/master/src/javascript/ContentEditor/DesignSystem/ReferenceCard/ReferenceCard.jsx#L195-L200

Reply:
Yes, we will add the ability to manage the icon for the EmptyCardSelector.